README.md

```diff
@@ -694,6 +694,35 @@ _NOTE:_ The field-level setting overrides the global config setting (for the fie
 
 </details>
 
+<details>
+<summary>Discarding nil fields</summary>
+
+
+By default nil values are included. You can override this behavior by passing `:discard_nil` option.
+
+Usage:
+
+```ruby
+class UserBlueprint < Blueprinter::Base
+  identifier :uuid
+
+  field :name
+  field(:birthday, discard_nil: true) do |_user, options|
+    nil
+  end
+end
```

Comment:
Let's simplify this a bit:

```suggestion
Usage:

```ruby
class UserBlueprint < Blueprinter::Base
  identifier :uuid

  field :name
  field :birthday, exclude_if_nil: true
end

user = User.new(name: 'John Doe')
puts UserBlueprint.render(user)
```